inst/extdata/level-of-service-table-complete.csv

```diff
@@ -1,113 +1,113 @@
-40 mph,1000 to 1999,Off road cycleway,3
-40 mph,1000 to 1999,Painted Cycle Lane,1
-40 mph,1000 to 1999,Segregated Track (narrow),1
-40 mph,1000 to 1999,Segregated Track (wide),1
-40 mph,2000 to 3999,Mixed Traffic Street,0
-40 mph,2000 to 3999,Painted Cycle Lane,1
-40 mph,2000 to 3999,Segregated Track (narrow),1
-40 mph,2000 to 3999,Segregated Track (wide),1
-40 mph,4000+,Mixed Traffic Street,0
-40 mph,4000+,Painted Cycle Lane,1
-40 mph,4000+,Segregated Track (narrow),1
-40 mph,4000+,Segregated Track (wide),1
-50 mph,0 to 999,Mixed Traffic Street,1
-50 mph,0 to 999,Off road cycleway,3
-50 mph,0 to 999,Painted Cycle Lane,1
-50 mph,0 to 999,Segregated Track (narrow),1
-50 mph,0 to 999,Segregated Track (wide),1
-50 mph,1000 to 1999,Mixed Traffic Street,0
-50 mph,1000 to 1999,Off road cycleway,3
-50 mph,1000 to 1999,Painted Cycle Lane,0
-50 mph,1000 to 1999,Segregated Track (narrow),0
-50 mph,1000 to 1999,Segregated Track (wide),1
-50 mph,2000 to 3999,Mixed Traffic Street,0
-50 mph,2000 to 3999,Painted Cycle Lane,0
-50 mph,2000 to 3999,Segregated Track (narrow),0
-50 mph,2000 to 3999,Segregated Track (wide),1
-50 mph,4000+,Mixed Traffic Street,0
-50 mph,4000+,Painted Cycle Lane,0
-50 mph,4000+,Segregated Track (narrow),0
-50 mph,4000+,Segregated Track (wide),1
-60+ mph,0 to 999,Mixed Traffic Street,0
-60+ mph,0 to 999,Off road cycleway,3
-60+ mph,0 to 999,Painted Cycle Lane,0
-60+ mph,0 to 999,Segregated Track (narrow),0
-60+ mph,0 to 999,Segregated Track (wide),1
-60+ mph,1000 to 1999,Mixed Traffic Street,0
-60+ mph,1000 to 1999,Painted Cycle Lane,0
-60+ mph,1000 to 1999,Segregated Track (narrow),0
-60+ mph,1000 to 1999,Segregated Track (wide),1
-60+ mph,2000 to 3999,Mixed Traffic Street,0
-60+ mph,2000 to 3999,Painted Cycle Lane,0
-60+ mph,2000 to 3999,Segregated Track (narrow),0
-60+ mph,2000 to 3999,Segregated Track (wide),1
-60+ mph,4000+,Mixed Traffic Street,0
-60+ mph,4000+,Painted Cycle Lane,0
-60+ mph,4000+,Segregated Track (narrow),0
-60+ mph,4000+,Segregated Track (wide),1
-<20 mph,0 to 999,Mixed Traffic Street,3
-<20 mph,0 to 999,Off road cycleway,3
-<20 mph,0 to 999,Painted Cycle Lane,3
-<20 mph,0 to 999,Segregated Track (narrow),3
-<20 mph,0 to 999,Segregated Track (wide),3
-<20 mph,1000 to 1999,Mixed Traffic Street,3
-<20 mph,1000 to 1999,Off road cycleway,3
-<20 mph,1000 to 1999,Painted Cycle Lane,3
-<20 mph,1000 to 1999,Segregated Track (narrow),3
-<20 mph,1000 to 1999,Segregated Track (wide),3
-<20 mph,2000 to 3999,Mixed Traffic Street,2
-<20 mph,2000 to 3999,Off road cycleway,3
-<20 mph,2000 to 3999,Painted Cycle Lane,3
-<20 mph,2000 to 3999,Segregated Track (narrow),3
-<20 mph,2000 to 3999,Segregated Track (wide),3
-<20 mph,4000+,Mixed Traffic Street,1
-<20 mph,4000+,Off road cycleway,3
-<20 mph,4000+,Painted Cycle Lane,2
-<20 mph,4000+,Segregated Track (narrow),2
-<20 mph,4000+,Segregated Track (wide),3
+Speed Limit (mph),Speed Limit (kph),AADT,cycle_segregation,level_of_service
+20 mph,32 kph,0 to 999,Mixed Traffic Street,3
```

Comment:
More relevant for the CbD is the assumed value of a "Speed band kph (85th percentile)" column that corresponds to the following:

```
20 mph -> 0 kph to 30 kph
30 mph ->  30 kph to 50 kph
40 mph -> 50 kph to 65 kph
50 mph -> 65 kph to 80 kph
60mph -> 80 kph to 95 kph
60+ mph -> 95 kph to 110 kph
```

Can you add an additional "Speed band kph (85th percentile)" column?